media-sound/bitwig-studio/bitwig-studio-1.3.16.ebuild

```diff
@@ -0,0 +1,90 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+# TODO
+# - Verify all runtime dependencies, min versions, and required flags
```

Comment:
I think we can remove this one, right?
With the current DEPENDS it works as it should, don't think we need specific versions at this point, right?